PointOfDispensingApp/src/components/VaccinationProceedComponent.vue

```diff
@@ -264,6 +266,12 @@ export default {
       this.$store.dispatch("patientDischarged", payload);
     },
     submitVaccinationRecord() {
+      this.$refs.form.validate();
```

Comment:
This _works_ in that it validates/indicates invalid fields after a submission attempt, but does it make it more obvious to the user to validate before the confirming "Are you sure?" rather than after (like warning/the field validation on attempting to click "Submit Vaccination Record")

Reply:
Good point. I've implemented this